### PR TITLE
Feature/149 new content page

### PR DESCRIPTION
**Pull request by VeronicaM**
Closes https://github.com/code4romania/ce-ma-fac/issues/149
Closes https://github.com/code4romania/ce-ma-fac/issues/168

Please mention the main changes this PR brings.

- There was no design available for this new page about coronavirus myths

### What is this change
- stored myth [images]( https://drive.google.com/open?id=1S6Q2hw3cJbwbYqYuD8c3sj5bL0-fFrgR) in `src/images/mituri` folder
-  new page of accordion type with all the content from the [spreadsheet] (https://docs.google.com/spreadsheets/d/1MPtwc03rTIWn1MguuFQPOrm1jhO-j_kd7rX7E3fd96g/edit?usp=sharing)

### CODE REVIEW

**Comment by moonflare** [comment]
Hey @VeronicaM Can you please do a rebase from develop?

File: frontend/src/styles/styles.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default {
```
Comment (moonflare) [inline frontend/src/styles/styles.js]: This is what we would usually do and I like it, but in the future we want the styles to come with the static content. So we should have them in the static content for now. 🙄

**Comment by aniri** [comment]
also @VeronicaM some small issues were reported here https://github.com/code4romania/ce-ma-fac/issues/168 ? could you please take a look?

**Comment by VeronicaM** [comment]
> Hey @VeronicaM Can you please do a rebase from develop?

Yep, forgot to do push --force :D

**Comment by VeronicaM** [comment]
> also @VeronicaM some small issues were reported here #168 ? could you please take a look?

Thanks, This should be fixed now

**Comment by aniri** [comment]
hey @moonflare @VeronicaM yep, I'll ask for a final check and approval on the content. we should also check that the images are compressed :smile:

**Comment by moonflare** [comment]
> hey @moonflare @VeronicaM yep, I'll ask for a final check and approval on the content. we should also check that the images are compressed 😄

We (I) can open a PR with the compressed images once this PR is fixed 😄

**Comment by aniri** [comment]
@moonflare @VeronicaM we got approval on the content :tada: merging this into develop :D

**Pull request merged**